### PR TITLE
Fixes Parameter style property getter that changes model data 

### LINKED ISSUE #1132: Property getter that changes model data is not the best way (imho)
Version 1.4.5:

Reading the value of OpenApiParameter.Style can change the value (Line 80 - SetDefaultStyleValue):
grafik

A member starting with "_" is public (Line 20):
grafik

It was not easy to find out, why my code does not work as expected and what I have to do.

### PR DESCRIPTION
**Pull request by MaggieKimani1**
Fixes #1132

### CODE REVIEW

File: src/Microsoft.OpenApi/Models/OpenApiParameter.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ namespace Microsoft.OpenApi.Models
     public class OpenApiParameter : IOpenApiSerializable, IOpenApiReferenceable, IEffective<OpenApiParameter>, IOpenApiExtensible
     {
         private bool? _explode;
-        public ParameterStyle? _style;
+        private ParameterStyle? _style;
```
Comment (baywet) [inline src/Microsoft.OpenApi/Models/OpenApiParameter.cs:20]: this is a breaking change, the PR should be rebased on v2

**Review by baywet — CHANGES_REQUESTED** [review]

File: src/Microsoft.OpenApi/Models/OpenApiParameter.cs (hunk 1 above)
Comment (MaggieKimani1) [reply src/Microsoft.OpenApi/Models/OpenApiParameter.cs:20]: Done.

Comment (baywet) [reply src/Microsoft.OpenApi/Models/OpenApiParameter.cs:20]: I think something went wrong with the rebase, it's bringing way more commits than it should. If we need to refresh the v2 branch first, let's do it as a separate PR if you don't mind. Let me know if you need help

**Comment by baywet** [comment]
@MaggieKimani1 If you need help with the git foo

To get just your commits on a branch to PR the 2.X branch:

```bash
git switch release/2.0.0
git pull -p
git switch -c bugfix/style
git cherry-pick 5b47268d3cc4d519bb6f789995d8ade87cbe5130..982c786272bb557a75844ccdf6bc3ebe28c1667b
gh pr create -f -B release/2.0.0
```

To refresh the 2.X branch with the latest changes from vnext

```bash
git switch vnext
git pull -p
git switch release/2.0.0
git pull
git switch -c feature/vnext-refresh
git merge vnext #you might have to handle conflicts
gh pr create -f -B release/2.0.0
```

**Comment by MaggieKimani1** [comment]
> @MaggieKimani1 If you need help with the git foo
> 
> To get just your commits on a branch to PR the 2.X branch:
> 
> ```shell
> git switch release/2.0.0
> git pull -p
> git switch -c bugfix/style
> git cherry-pick 5b47268d3cc4d519bb6f789995d8ade87cbe5130..982c786272bb557a75844ccdf6bc3ebe28c1667b
> gh pr create -f -B release/2.0.0
> ```
> 
> To refresh the 2.X branch with the latest changes from vnext
> 
> ```shell
> git switch vnext
> git pull -p
> git switch release/2.0.0
> git pull
> git switch -c feature/vnext-refresh
> git merge vnext #you might have to handle conflicts
> gh pr create -f -B release/2.0.0
> ```

Thank you for this.

**Pull request closed without merging**